test/functional/core/job_spec.lua

```diff
@@ -260,41 +301,39 @@ describe('jobs', function()
   end)
 
   it('can get the pid value using getpid', function()
-    if helpers.pending_win32(pending) then return end  -- TODO: Need `cat`.
-    nvim('command', "let j =  jobstart(['cat', '-'], g:job_opts)")
+    nvim('command', "let j =  jobstart(has('win32') ? ['find', '/v', ''] : ['cat', '-'], g:job_opts)")
     local pid = eval('jobpid(j)')
-    eq(0,os.execute('ps -p '..pid..' > /dev/null'))
+    eq(0, os_ps(pid))
     nvim('command', 'call jobstop(j)')
     eq({'notification', 'stdout', {0, {''}}}, next_msg())
-    eq({'notification', 'exit', {0, 0}}, next_msg())
-    neq(0,os.execute('ps -p '..pid..' > /dev/null'))
+    eq({'notification', 'exit', {0, iswin() and 1 or 0}}, next_msg())
+    neq(0, os_ps(pid))
   end)
 
   it("do not survive the exit of nvim", function()
-    if helpers.pending_win32(pending) then return end
     -- use sleep, which doesn't die on stdin close
-    nvim('command', "let g:j =  jobstart(['sleep', '1000'], g:job_opts)")
+    nvim('command', "let g:j =  jobstart(has('win32') ? ['ping', '-n', '1001', '127.0.0.1'] : ['sleep', '1000'], g:job_opts)")
     local pid = eval('jobpid(g:j)')
-    eq(0,os.execute('ps -p '..pid..' > /dev/null'))
+    eq(0, os_ps(pid))
     clear()
-    neq(0,os.execute('ps -p '..pid..' > /dev/null'))
+    neq(0, os_ps(pid))
   end)
 
   it('can survive the exit of nvim with "detach"', function()
-    if helpers.pending_win32(pending) then return end
     nvim('command', 'let g:job_opts.detach = 1')
-    nvim('command', "let g:j = jobstart(['sleep', '1000'], g:job_opts)")
+    nvim('command', "let g:j = jobstart(has('win32') ? ['ping', '-n', '1001', '127.0.0.1'] : ['sleep', '1000'], g:job_opts)")
     local pid = eval('jobpid(g:j)')
-    eq(0,os.execute('ps -p '..pid..' > /dev/null'))
+    eq(0, os_ps(pid))
     clear()
-    eq(0,os.execute('ps -p '..pid..' > /dev/null'))
+    eq(0, os_ps(pid))
     -- clean up after ourselves
-    os.execute('kill -9 '..pid..' > /dev/null')
+    os_kill(pid)
+    neq(0, os_ps(pid))
```

Comment:
It should be NOT found since we killed it.

Reply:
I'm  checking if the `os_kill` did work. Is there a better way to do it or do you want to check the exit code of `os_kill`?

Reply:
@janlazo Ah, I didn't realize `os_ps` was returned the exit code, I thought it returned a `pid`. Anyways `nvim_get_proc` should be used instead.